shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -461,6 +619,9 @@ - (void)testCalculateKeyboardInset {
   CGRect convertedViewFrame = CGRectMake(20, 20, 320, screenHeight - 40);
   CGRect keyboardFrame = CGRectMake(20, screenHeight - 320, screenWidth, 300);
 
+  [self setupMockMainScreenAndView:viewControllerMock
+                         viewFrame:viewOrigFrame
+                    convertedFrame:convertedViewFrame];
   id mockView = OCMClassMock([UIView class]);
```

Comment:
It won't let me comment on the line, but lines 614 and lines 625-629 can be removed since you're using `setupMockMainScreenAndView`.

```
614  OCMStub([viewControllerMock mainScreenIfViewLoaded]).andReturn(UIScreen.mainScreen);
```
```
625  id mockView = OCMClassMock([UIView class]);
626  OCMStub([mockView frame]).andReturn(viewOrigFrame);
627  OCMStub([mockView convertRect:viewOrigFrame toCoordinateSpace:[OCMArg any]])
628      .andReturn(convertedViewFrame);
629  OCMStub([viewControllerMock viewIfLoaded]).andReturn(mockView);